tests/test_iqm_operation_mapping.py

```diff
@@ -114,3 +114,8 @@ def test_instruction_to_operation():
     operation = instruction_to_operation(instruction)
     assert isinstance(operation.gate, IQMMoveGate)
     assert operation.qubits == (cirq.NamedQubit('QB1'), cirq.NamedQid('COMP_R', dimension=2))
+
+    instruction = Instruction(name='reset', qubits=('QB1',))
+    operation = instruction_to_operation(instruction)
+    assert isinstance(operation.gate, ResetChannel)
+    assert operation.qubits == (cirq.NamedQubit('QB1'))
```

Comment:
There is no reverse test case? Operation to instruction?

Reply:
If not, it ought to be added.